includes/Integrations/Site_Kit.php

```diff
@@ -0,0 +1,167 @@
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\Integrations;
+
+use Google\Web_Stories\Analytics;
+use Google\Web_Stories\Story_Post_Type;
+
+/**
+ * Class Site_Kit.
+ */
+class Site_Kit {
+	/**
+	 * Analytics instance.
+	 *
+	 * @var Analytics
+	 */
+	protected $analytics;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Analytics $analytics Analytics instance.
+	 */
+	public function __construct( Analytics $analytics ) {
+		$this->analytics = $analytics;
+	}
+
+	/**
+	 * Initializes all hooks.
+	 *
+	 * @since 1.2.0
+	 *
+	 * @return void
+	 */
+	public function init() {
+		add_filter( 'googlesitekit_amp_gtag_opt', [ $this, 'filter_site_kit_gtag_opt' ] );
+
+		if ( in_array( 'analytics', $this->get_site_kit_active_modules_option(), true ) ) {
+			remove_action( 'web_stories_print_analytics', [ $this->analytics, 'print_analytics_tag' ] );
+		}
+	}
+
+	/**
+	 * Filters Site Kit's Google Analytics configuration.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param array $gtag_opt Array of gtag configuration options.
+	 *
+	 * @return array Modified configuration options.
+	 */
+	public function filter_site_kit_gtag_opt( $gtag_opt ) {
+		if ( ! is_singular( Story_Post_Type::POST_TYPE_SLUG ) ) {
+			return $gtag_opt;
+		}
+
+		$default_config             = $this->analytics->get_default_configuration( $gtag_opt['vars']['gtag_id'] );
+		$default_config['triggers'] = isset( $default_config['triggers'] ) ? $default_config['triggers'] : [];
+
+		$gtag_opt['triggers'] = isset( $gtag_opt['triggers'] ) ? $gtag_opt['triggers'] : [];
+		$gtag_opt['triggers'] = array_merge(
+			$default_config['triggers'],
+			$gtag_opt['triggers']
+		);
+
+		return $gtag_opt;
+	}
+
+	/**
+	 * Gets the option containing the active Site Kit modules.
+	 *
+	 * Checks two options as it was renamed at some point in Site Kit.
+	 *
+	 * Bails early if the Site Kit plugin itself is not active.
+	 *
+	 * @see \Google\Site_Kit\Core\Modules\Modules::get_active_modules_option
+	 *
+	 * @since 1.2.0
+	 *
+	 * @return array List of active module slugs.
+	 */
+	private function get_site_kit_active_modules_option() {
+		if ( ! defined( 'GOOGLESITEKIT_VERSION' ) ) {
+			return [];
+		}
+
+		$option = get_option( 'googlesitekit_active_modules' );
+
+		if ( is_array( $option ) ) {
+			return $option;
+		}
+
+		$legacy_option = get_option( 'googlesitekit-active-modules' );
+
+		if ( is_array( $legacy_option ) ) {
+			return $legacy_option;
+		}
+
+		return [];
+	}
+
+	/**
+	 * Returns the Site Kit plugin status.
+	 *
+	 * @since 1.2.0
+	 *
+	 * @return array Plugin status.
+	 */
+	public function get_plugin_status() {
+		$is_installed        = array_key_exists( 'google-site-kit/google-site-kit.php', get_plugins() );
+		$is_active           = defined( 'GOOGLESITEKIT_VERSION' );
+		$is_analytics_active = in_array( 'analytics', $this->get_site_kit_active_modules_option(), true );
+
+		$link      = __( 'https://wordpress.org/plugins/google-site-kit/', 'web-stories' );
```

Comment:
Would a translation comment be useful here?

Reply:
Not really, not much more context could be provided here. The URL itself is pretty self-explanatory.